routers/api/v1/repo/issue_subscription.go

```diff
@@ -278,10 +278,17 @@ func GetIssueSubscribers(ctx *context.APIContext) {
 		ctx.Error(http.StatusInternalServerError, "GetUsersByIDs", err)
 		return
 	}
-	apiUsers := make([]*api.User, 0, len(users))
-	for i := range users {
-		apiUsers[i] = convert.ToUser(users[i], ctx.User)
+	var apiUsers []*api.User
+	for _, v := range users {
+		apiUsers = append(apiUsers, convert.ToUser(v, ctx.User))
```

Comment:
```suggestion
	apiUsers := make([]*api.User, 0, len(users))
	for _, v := range users {
		apiUsers = append(apiUsers, convert.ToUser(v, ctx.User))
```

Reply:
How could the old code work? It should have crashed on the first write.

Reply:
Golang allow that after some version.